### ISSUE TITLE
 Chronograf (1.8.0) Cannot read property 'name' of undefined

### ISSUE THREAD

**Issue by SteverInAZ** [influxdata/chronograf#5404]
What browser are you using (name and version)?

What operating system are you using?

Please describe what you were trying to do when you encountered this error:

```
TypeError: Cannot read property 'name' of undefined
    at http://172.16.18.62:8888/src.5692061f.js:2365:385
    at Array.map (<anonymous>)
    at l (http://172.16.18.62:8888/src.5692061f.js:2365:310)
    at sa (http://172.16.18.62:8888/src.5692061f.js:717:55869)
    at $a (http://172.16.18.62:8888/src.5692061f.js:717:68127)
    at oo (http://172.16.18.62:8888/src.5692061f.js:717:91055)
    at uo (http://172.16.18.62:8888/src.5692061f.js:717:91442)
    at Yo (http://172.16.18.62:8888/src.5692061f.js:717:98524)
    at $o (http://172.16.18.62:8888/src.5692061f.js:717:97904)
    at eu (http://172.16.18.62:8888/src.5692061f.js:717:99279)
```

**Comment by russorat** [comment]
@SteverInAZ thanks for opening. Could you answer the following questions to help us debug?

What browser are you using (name and version)?
What operating system are you using?
Please describe what you were trying to do when you encountered this error:

**Comment by glinton** [comment]
This should be fixed by #5420 and will be available in 1.8.1
